src/shogun/kernel/GaussianARDKernel.h

```diff
@@ -4,25 +4,60 @@
  * the Free Software Foundation; either version 3 of the License, or
  * (at your option) any later version.
  *
+ * Written (W) 2015 Wu Lin
  * Written (W) 2012 Jacob Walker
  *
  * Adapted from WeightedDegreeRBFKernel.h
  */
 
-#ifndef GAUSSIANARDKERNEL_H_
-#define GAUSSIANARDKERNEL_H_
+#ifndef _GAUSSIANARDKERNEL_H_
+#define _GAUSSIANARDKERNEL_H_
```

Comment:
@karlnapf umm I remember you suggesting something regarding the usage of underscores in the beginning of these. @yorkerlin is there any particular reason you changed this?